docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>Ha, I should read it completely before writing because above I basically repeated what you say here. I still have some comments thought.</p><p> </p><p>Regarding the first sentence. I think the "adjusted" in "adjusted predictions" comes from the fact that you're "adjusting for covariates" when computing the predictions. This is also seen as "controlling for the covariates" but the latter is a bit more misleading because it may be interpreted that you controlled the covariates and that's not always the case. </p><p> </p><p>I think this is what you say in the last sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<blockquote>Regarding the first sentence. I think the "adjusted" in "adjusted predictions" comes from the fact that you're "adjusting for covariates" when computing the predictions</blockquote><p>This is also my interpretation also. I have adjusted the wording to include  "adjusted predictions refers to the outcome predicted by a fitted model on a specified scale for a given combination of values of the predictor variables, such as their observed values, their means, or some user specified grid of values". Regarding the "conditional" part, I now explicitly state it is the "specific predictor(s) we would like to condition on...."</p>